hybrid/compartment_examples/inter_comp_call/clean_leak_ddc.s

```diff
@@ -0,0 +1,41 @@
+// A broken version of `clean.S`, which simulates leaking the address of
+// switcher DDC
+clean:
+    mov x0, #0
+    mov x1, #0
+    mov x2, #0
+    mov x3, #0
+    mov x4, #0
+    mov x5, #0
+    mov x6, #0
+    mov x7, #0
+    mov x8, #0
+    mov x9, #0
+    mov x10, #0
+    mov x11, #0
+    mov x12, #0
+    mov x13, #0
+    mov x14, #0
+    mov x15, #0
+    mov x16, #0
+    mov x17, #0
+    // x18 is the "platform register" (for some platforms). If so, it needs to
+    // be preserved, but here we assume that only the lower 64 bits are
+    // required.
+    mov x18, x18
+    // x19-x29 are callee-saved, but only the lower 64 bits.
+    mov x19, x19
+    // Simulate leaking address of switcher DDC
```

Comment:
If I read the tree correctly, this will be going in the "root" folder, rather than in one specific example directory. However, this comment, I believe, corresponds to one of the particular examples, where we chose `c20` to be the target for DDC switching. I would perhaps rephrase this comment to something like "simulate leaking a register, used to evaluate security issues".

Reply:
Addressed in 2edd256406a0480c26bb8051a33c7ba53fdc755a